package.json

```diff
@@ -38,6 +39,7 @@
     "jest": "^23.5.0",
     "jest-tap-reporter": "^1.9.0",
     "node-mocks-http": "^1.7.0",
+    "nodemon": "^1.18.3",
```

Comment:
Are we using **nodemon** somewhere?
I didn't see it in any npm scripts or used anywhere specifically.

Reply:
I started using it locally. This PR includes a new `nodemonConfig` section in `package.json` so we can use `nodemon server.js` to automatically refresh the `server.js` process when files are changed. I'll add it to the README>